FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.SchemaManagement/Parquet/SchemaProvider/JsonSchemaParser.cs

```diff
@@ -18,14 +18,14 @@ public class JsonSchemaParser
         /// <param name="resourceType">The resource type.</param>
         /// <param name="jsonSchema">The json schema.</param>
         /// <returns>A FhirParquetSchemaNode instance.</returns>
-        public static FhirParquetSchemaNode ParseJSchema(string resourceType, JSchema jsonSchema)
+        public static FhirParquetSchemaNode ParseJSchema(string resourceType, JsonSchema jsonSchema)
         {
-            if (jsonSchema.Type == null || jsonSchema.Type == JSchemaType.Null)
+            if (jsonSchema.Type == JsonObjectType.Null)
```

Comment:
is it possible that jsonSchema.Type is null?

Reply:
Updated, now will check **JSchemaType.None** or **JSchemaType.Null**, if no raw value exists, the JsonSchema parser will assign **JSchemaType.None** for the schema type, thanks.